### ISSUE TITLE
Add Sparse Softmax Cross Entropy loss function

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#830]
Copied from original issue https://github.com/tensorflow/tfjs/issues/131

**Comment by euler16** [comment]
I want to work on this issue (if no one else is currently working on this). @manrajgrover could you assign me this

**Comment by rthadur** [comment]
This has been implemented here , so closing this issue.

**Comment by iislucas** [comment]
I actually don't I don't see it in the tfjs APIs... does anyone know where to find it?
